lib/active_model/serializer.rb

```diff
@@ -50,6 +50,10 @@ def self.type(type)
       self._type = type
     end
 
+    def self.links(&block)
+      self._links_block = block
+    end
```

Comment:
Alternative? You actually need this block every time to instantiate a serializer.